src/main/java/org/jitsi/videobridge/rest2/Conferences.java

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
This commit just says "rename variable" but it's also adding this file...

Reply:
good catch, fixed